qiskit_metal/qlibrary/core/design_check.py

```diff
@@ -0,0 +1,86 @@
+### BASIC PROGRAM THAT LOOKS FOR OVERLAP BETWEEN QCOMPONENTS
+from numpy import size
+import geopandas as gpd
+import pandas as pd
+
+
+class QDesignCheck():
+    """QDesign_Check contains various design checks, such as
+    testing designs in qiskit metal for unintended overlap
+    between components and/or connections between components.
+    """
+
+    def __init__(self, design: 'QDesign'):
+        self.design = design
+
+    def update_design(self, design: 'QDesign'):
+        self.design = design
+
+    def overlap_tester(self):
+        design = self
+        """This particular function tests for overlap amongst qcomponents
+        and CPWs. It will change qubit/qubit overlap, qubit, CPW overlap
```

Comment:
"It will change qubit/qubit overlap..."  

Guessing meant "check" instead of "changed"?, not to be nitpicky

Reply:
I actually meant to write "catch" instead of "change." Updated accordingly. Good catch (pun intended!)